specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/virtualWan.json

```diff
@@ -4204,10 +4672,164 @@
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "connection type."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Virtual Appliance Site resource."
+    },
+    "BgpConnectionProperties": {
+      "properties": {
+        "peerAsn": {
+          "type": "integer",
+          "readOnly": false,
+          "format": "int64",
+          "minimum": 0,
+          "maximum": 4294967295,
+          "description": "Peer ASN."
+        },
+        "peerIp": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Peer IP."
+        },
+        "provisioningState": {
+          "description": "The provisioning state of the resource.",
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState"
+        },
+        "connectionState": {
+          "type": "string",
+          "description": "The current state of the VirtualHub to Peer.",
+          "readOnly": true,
+          "enum": [
+            "Unknown",
+            "Connecting",
+            "Connected",
+            "NotConnected"
+          ],
+          "x-ms-enum": {
+            "name": "HubBgpConnectionStatus",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Properties of the bgp connection."
+    },
+    "HubIpConfiguration": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/HubIPConfigurationPropertiesFormat",
+          "description": "The properties of the Virtual Hub IPConfigurations."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the Ip Configuration."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Ipconfiguration type."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "IpConfigurations"
+    },
+    "HubIPConfigurationPropertiesFormat": {
+      "properties": {
+        "privateIPAddress": {
+          "type": "string",
+          "description": "The private IP address of the IP configuration."
+        },
+        "privateIPAllocationMethod": {
+          "$ref": "./network.json#/definitions/IPAllocationMethod",
+          "description": "The private IP address allocation method."
+        },
+        "subnet": {
+          "$ref": "./virtualNetwork.json#/definitions/Subnet",
+          "description": "The reference to the subnet resource."
```

Comment:
do we need specific name for the subnet?

Reply:
No. This is same as IpConfiguration in networkInterface.json. Subnet is generic name and we would need it to be as it is